packages/typeahead/src/react/index.tsx

```diff
@@ -237,9 +231,14 @@ const Typeahead: TypeaheadFieldComponent = props => {
   return (
     <BelowLeft
       show={
-        <div {...styles.wrapper(isOpen)}>
+        <div
+          className={classNames(
+            'psds-multi-select__wrapper',
+            isOpen && 'psds-multi-select__wrapper--open'
+          )}
+        >
           <Menu
-            {...styles.menu()}
+            className="psds-multi-select__menu"
```

Comment:
Please add className prop composition to parent node of exposed components.

Reply:
Not sure that applies here, due to how we designed Position quite frankly. If I had made this thing it would of been a hook not a jsx Element and not something that wrapped what I see as the root node. From a render standpoint Field is the root and thus it already receives the rest spread.

Reply:
I think the argument about choosing a different root node is a good one. Sometimes it's not super obvious. If it's Field, I'm good with that. Maybe I'm misreading this instance. In general, I think the parent node is usually the most worth while, because css selectors can have child selectors but not parent selectors.